### DISCUSSION TITLE
Media Queries not working well

### DISCUSSION THREAD

**Discussion by nimsara-c** [tailwindlabs/tailwindcss#5852] [Help] [answered]
Media Queries in TailWind CSS are not working Well.<br/>

*My Code is this*
``` html
<img src={MYIMAGE} alt="Test Image" className="h-6 w-8 rounded-md gfold:h-8 gfold:w-14 gfold:rounded-xl sm:h-16 sm:w-24 " />
```
`gfold` -> `(400px)` - A Media Size included by me in the `tailwind.config.js` file. <br/>
`sm` -> `(640px)` - A Default Media Size that Comes with Tailwind.

This is the Code in [Tailwind Play](https://play.tailwindcss.com/KAYPfXUSwO?size=332x720)

When Media Size is below `sm` size is correct, But when my screen is greater than the `sm` size the ***Size is not Changing***.

**I want to Change the Image Size when the Screen Size Changes**

Now what should I do? *Please help me*, I am new to Tailwind CSS.

**Comment by adamwathan** [comment, marked as answer]
Hey! See this section of the docs, it explains what you need to do to add an additional small breakpoint:

https://tailwindcss.com/docs/breakpoints#extending-the-default-breakpoints

**Reply by nimsara-c** [reply]
Thank You So Much
